ngraph/frontend/cmake_static_protobuf/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13)
+
+project(libprotobuf_static)
+
+message("Add PROTOBUF dependency - static")
+
+set(BUILD_SHARED_LIBS_OLD ${BUILD_SHARED_LIBS})
+set(BUILD_STANDALONE_STATIC_OLD ${BUILD_STANDALONE_STATIC})
+set(USE_STATIC_PROTOBUF_OLD ${USE_STATIC_PROTOBUF})
+
+set(BUILD_SHARED_LIBS OFF)
+set(BUILD_STANDALONE_STATIC ON)
+set(USE_STATIC_PROTOBUF ON)
+
+include(../../cmake/external_protobuf.cmake)
```

Comment:
Why do you use relative path?
And why do you need to have static protobuf? Why cannot we use one protobuf for all OV components?

Reply:
See my response in comment above

Reply:
Regarding 'relative path' - static protobuf is currently planned to be used by frontends, thus it is placed inside "frontend" folder. It needs to include "external_protobuf.cmake" which is "ngraph/cmake" folder